content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -221,71 +221,107 @@ sysctl --system
-```
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Debian Unstable  | `Debian_Unstable` |
+| Debian Testing   | `Debian_Testing`  |
+<br />
+Then, set `$VERSION` to be the cri-o version matching your kubernetes version.
+For instance, if you want to install cri-o 1.18, `VERSION=1.18`.
+We also support pinning to a particular release. To install 1.18.3, `VERSION=1.18:1.18.3`.
+<br />
 
+Then run
 ```shell
-# Debian 10
-echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_10/Release.key -O- | sudo apt-key add -
-```
+echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
 
-```shell
-# Raspbian 10
-echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Raspbian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Raspbian_10/Release.key -O- | sudo apt-key add -
-```
+curl -L https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/Release.key | apt-key add -
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | apt-key add -
 
-and then install CRI-O:
-```shell
-sudo apt-get install cri-o-1.17
+apt-get update
+apt-get install cri-o cri-o-runc
 ```
+
 {{% /tab %}}
 
-{{% tab name="Ubuntu 18.04, 19.04 and 19.10" %}}
+{{% tab name="Ubuntu" %}}
 
-```shell
-# Configure package repository
-. /etc/os-release
-sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/x${NAME}_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
-wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/x${NAME}_${VERSION_ID}/Release.key -O- | sudo apt-key add -
-sudo apt-get update
-```
+To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
 
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Ubuntu 20.04     | `xUbuntu_20.04`   |
+| Ubuntu 19.10     | `xUbuntu_19.10`   |
+| Ubuntu 19.04     | `xUbuntu_19.04`   |
+| Ubuntu 18.04     | `xUbuntu_18.04`   |
+<br />
+Then, set `$VERSION` to be the cri-o version matching your kubernetes version.
+For instance, if you want to install cri-o 1.18, `VERSION=1.18`.
+We also support pinning to a particular release. To install 1.18.3, `VERSION=1.18:1.18.3`.
```

Comment:
nit: There are several places where you use the phrase, "We also support pinning".
What do you think about rewording the phrase to something like:
CRI-O supports pinning to a specific release.
For example, to install version 1.18.3 set `VERSION=1.18:1.18.3`.
OR
You can pin your installation to a specific release. To install version 1.18.3, set `VERSION=1.18:1.18.3`.